### ISSUE TITLE
Panels are broken in MATE.

### ISSUE THREAD

**Issue by mknepper** [grassmunk/Chicago95#46]
If you use Chicage95 in Ubuntu MATE 18.04, or 18.10, the panel borders are broken - they don't render unless it's directly under an active menu. I'll attach a picture:

![brokenpanels](https://user-images.githubusercontent.com/12740958/47808333-24b52f00-dd0c-11e8-8552-b9494a683f43.png)

This is a real shame because everything else looks pretty good! I'm not sure why it's doing this, but the devs at MATE did manage to get borders working on their themes. See this: [https://github.com/mate-desktop/mate-panel/issues/344#issuecomment-434220815](url)

**Comment by mknepper** [comment]
So far it seems to be adding some CSS in the gtkwidgets.css or whatever it's called to fix it. However you set the borders where the system tray and application menu are would probably need to be applied to the whole thing. I'd do it for you but I don't know where the border css is located.

**Comment by mknepper** [comment]
Will do! I'll fire up my Vbox and see how it works out. And yeah, I noticed MATE does weird stuff with its themes for panels. I'm not sure if this is the end result of forking from the original GTK2 or what. I'll have some screenshots and verify it works on Ubuntu MATE 18.04. I'll probably try it on Debian 9.x too.

**Comment by mknepper** [comment]
I should have test results for you in a day or two. Thanks again! :)

**Comment by mknepper** [comment]
![Screenshot at 2019-07-06 18-17-37](https://user-images.githubusercontent.com/12740958/60761991-94598780-a01a-11e9-85a7-3f0e1d295d92.png)

Happy to report that the issue has been solved! I'm not sure how 1 or 2 days turned into five months, but whatever. 18.04 LTS of Xubuntu. I installed mate-desktop-environment and tested it out. I can't 100% confirm it works on Ubuntu MATE, but it IS WORKING on the MATE desktop environment.
